sys/src/cmd/frontusb/lib/dump.c

```diff
@@ -12,10 +12,10 @@ static char* cnames[] =
 	"none", "audio", "comms", "hid", "",
 	"", "", "printer", "storage", "hub", "data"
 };
-static char* devstates[] =
+/*static char* devstates[] =
```

Comment:
why comment this out? usage warning?

Reply:
It's not used.  Can remove entirely tbh.